client/src/components/Explorer/ExplorerItem.scss

```diff
@@ -82,6 +89,11 @@
     font-size: 1.2em;
 }
 
+.icon--item-action {
+    width: 1em;
+    height: 1em;
+}
```

Comment:
This seems like a customisation of icons based on their context rather than a variation that will be reusable elsewhere, so I think it would be enough to target `.icon` within the context of this component.

Reply:
I agree in principle, however, in [this submenu item template context](https://github.com/wagtail/wagtail/pull/4821/files#diff-ca00f5418069bc4ee30e2434d64a0506), there are two icons within one `a.submenu-trigger` element: the item's unique icon, and the right arrow.

```html
    <a href="#" data-nav-primary-submenu-trigger class="submenu-trigger {{ classnames }}"{{ attr_string }}>
        {% if icon_name %}{% icon icon_name 'icon--menuitem' %}{% endif %}
        {{ label }}
        {% icon 'arrow-right' 'icon--submenu-trigger' %}
    </a>
```

I suppose they could be targeted with `:first-child` and `:last-child`, but that feels a little magical or too obfuscated to me, whereas it's easy to search by those class names to connect the Less to the template code, or vice versa.

Thoughts?